app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -149,8 +149,9 @@ def find_host_by_provider_id(provider_id)
     def scheme_to_provider_mapping
       @scheme_to_provider_mapping ||= begin
         {
-          'gce' => ['ManageIQ::Providers::Google::CloudManager'.safe_constantize, :name],
-          'aws' => ['ManageIQ::Providers::Amazon::CloudManager'.safe_constantize, :uid_ems],
+          'gce'   => ['ManageIQ::Providers::Google::CloudManager'.safe_constantize, :name],
+          'aws'   => ['ManageIQ::Providers::Amazon::CloudManager'.safe_constantize, :uid_ems],
+          'azure' => ['ManageIQ::Providers::Azure::CloudManager'.safe_constantize, :uid_ems],
```

Comment:
I want to get the UUID which is what is returned from here: https://github.com/google/cadvisor/blob/master/utils/cloudinfo/azure.go#L47

I couldn't test this yet because I don't have an azure machine.